tools/rostest/nodes/advertisetest

```diff
@@ -168,20 +166,20 @@ class AdvertiseTest(unittest.TestCase):
                     if s_name in s_name_set:
                         s_name_set.remove(s_name)
                         service = self.services[s_name]
-                        assert_true(t_elapsed < service['timeout'],
-                                    'Service [%s] is advertised before timeout [%s] secs' % (service['name'], service['timeout']))
-                        assert_false(service['negative'],
-                                     'Service [%s] is not advertised' % (service['name']))
+                        assert t_elapsed < service['timeout'], \
+                            'Service [%s] is advertised before timeout [%s] secs' % (service['name'], service['timeout'])
+                        assert not service['negative'], \
+                            'Service [%s] is not advertised' % (service['name'])
```

Comment:
Is the service not advertised, or unexpectedly advertised?

Reply:
I think this check ensure that such a a service is not advertised. Similar to the topic test above.

Reply:
> I think this check ensure that such a a service is not advertised.

It looks like if the test fails the output will be `AssertionError: Service [foo] is not advertised`. If that's what it's checking I think it should say the opposite.